docs/docs/sourcing-from-prismic.md

```diff
@@ -68,160 +60,142 @@ require("dotenv").config({
 module.exports = {
   plugins: [
     {
-      resolve: `gatsby-source-prismic`,
+      resolve: `gatsby-source-prismic-graphql`,
       options: {
         repositoryName: `your-repository-name`,
         accessToken: `${process.env.API_KEY}`,
-        linkResolver: ({ node, key, value }) => post => `/${post.uid}`,
-      },
+        pages: [
```

Comment:
How necessary is this pages array? I ask because the not-yet existent template references fail the `gatsby develop` command in the next section. It's also a departure from the previous guide, and the source plugin says it's optional. 

Because this is a guide rather than a tutorial, some options to resolve it could be to omit this array if unnecessary or move the GraphiQL explorer section to after the template creation section so the CLI command will run. That's rather than breaking up the configuration into steps, which also came to mind.